spinta/backends/postgresql/types/ref/init.py

```diff
@@ -29,10 +29,12 @@ def get_pg_foreign_key(
     *,
     table_name: str,
     model_name: str,
-    column_type: TypeEngine,
-) -> List[Union[sa.Column, sa.Constraint]]:
+    column_type: TypeEngine) -> List[Union[sa.Column, sa.Constraint]]:
     column_name = get_column_name(prop) + '._id'
     nullable = not prop.dtype.required
+    if prop.model.unique:
+        prop.model.unique = [list(map(lambda val: val.replace(
+            get_column_name(prop), column_name), val)) for val in prop.model.unique]
```

Comment:
We can't modify `model` here, this must be done in `load` and in load, we should deal with properties as instances of `Property` class. Here, we can only convert property names into backend naming without changing the existing metadata.